src/diff/index.js

```diff
@@ -60,7 +60,7 @@ export function diff(dom, parentDom, newVNode, oldVNode, context, isSvg, excessD
 
 				// If the last child is a Fragment, use _lastDomChild, else use _dom
 				p = newVNode._children[newVNode._children.length - 1];
-				newVNode._lastDomChild = p._lastDomChild || p._dom;
+				newVNode._lastDomChild = p && (p._lastDomChild || p._dom);
```

Comment:
Would this mean that, if my last vnode is falsy, it's gonna get skipped?

Reply:
We are targetting a vnode so only vnodes that evaluate to null are skipped. This is introduced in the `holes` PR. So '' --> gets coerced to vnode but null can be kept that's why this crashes null._lastDomChild or null._dom

Reply:
Thank you 🙂

Reply:
Thanks for this heads up, reminds me that I should always elaborate in my PR description instead of assuming that people also debugged it (which is a really strange thinking process I sometimes have)

Reply:
No worries 👍! I would've checked my assumption, first, but I was on my phone 😟